extensions/zen-browser/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Zen Changelog
 
+## [Fix Bookmark Duplicates Issue] - 2024-11-19
```

Comment:
This shouldn't include an actual date. This should have this placeholder `{PR_MERGE_DATE}`. This will be replaced later when the PR gets merged. https://developers.raycast.com/basics/prepare-an-extension-for-store#version-history

Reply:
My bad. I forgot about that.